runtime/flang/getvolinfo3f.c

```diff
@@ -16,26 +16,19 @@
 #include <string.h>
 
 typedef char *LPSTR;
+typedef const char *LPCSTR;
 typedef int DWORD;
+typedef DWORD *LPDWORD;
 
-#if defined(WIN64) || defined(WIN32)
-#if defined(WIN64)
-typedef long long LDWORD;
-extern int GetVolumeInformationA();
+#if defined(_WIN64)
+extern int GetVolumeInformationA(LPCSTR, LPSTR, DWORD, LPDWORD, LPDWORD,
```

Comment:
Is it not a better idea to `#include <fileapi.h>`?

Reply:
Hi Bryan, sorry for the delayed response, I've bumped into another issue with tip of tree Flang on Windows after the rebase.
Unfortunately  we cannot include <fileapi.h> directly. Basically 'fileapi' is automatically included in 'windows.h' and its dependencies as well. So this header is not a self-contained header because of its dependencies.

In this case we can only use 'windows.h' for example with WIN32_LEAN_AND_MEAN definition in order to exclude a bunch of unnecessary Windows header files.